stratum/tools/BUILD

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_pkg//:pkg.bzl", "pkg_deb", "pkg_tar")
+
+pkg_tar(
+    name = "stratum_tools_data",
+    srcs = [
+        "//stratum/hal/bin/barefoot:bf_pipeline_builder",
+        "//stratum/hal/bin/dummy:stratum_dummy",
+        "//stratum/hal/config:chassis_config_migrator",
+        "//stratum/hal/lib/phal:phal_cli",
+        "//stratum/hal/lib/phal/onlp:onlp_cli",
+        "//stratum/hal/stub/embedded:stratum_stub",
+        "//stratum/procmon:procmon_main",
+        "//stratum/tools/gnmi:gnmi_cli",
+        "//stratum/tools/stratum_replay",
+    ],
+    extension = "tar.xz",
```

Comment:
This gives us the best compression ratio and makes the whole package just 8.3M small.